pkg/semrel/github_test.go

```diff
@@ -0,0 +1,210 @@
+package semrel
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"regexp"
+	"testing"
+
+	"github.com/Masterminds/semver"
+	"github.com/google/go-github/v30/github"
+)
+
+func TestNewGitlabRepository(t *testing.T) {
```

Comment:
I think this should be called `TestNewGithubRepository`

Reply:
Whoops. Fixed